src/Backend/Modules/Analytics/GoogleClient/Connector.php

```diff
@@ -176,21 +176,22 @@ private function getData($startDate, $endDate)
     {
         $dateRange = $startDate . '-' . $endDate;
 
-        if (!$this->cache->isCached($dateRange)) {
-            $this->cache->cache(
-                $dateRange,
-                json_encode(
-                    array(
-                        'metrics' => $this->getMetrics($startDate, $endDate),
-                        'visitGraphData' => $this->collectVisitGraphData($startDate, $endDate),
-                        'pageViews' => $this->collectMostVisitedPagesData($startDate, $endDate),
-                        'sourceGraphData' => $this->collectSourceGraphData($startDate, $endDate),
-                    )
-                )
-            );
+        $item = $this->cache->getItem('analytics-'.$dateRange);
```

Comment:
We put spaces around our concat operator. (I know that PSR guidelines don't say anything about this, but it would be more conform with the Fork code base)